README.md

```diff
@@ -101,6 +101,32 @@ export type Locale = 'en' | 'br' | '<MY_NEW_LOCALE>';
 // Where <MY_NEW_LOCALE> is the new locale you added. For example, 'es', 'fr', 'de', etc.
 ```
 
+## How To Run
+### Locally
+
+To visualize your changes in another repository/project, execute the following commands:
+```
+npm run build
+```
+```
+npm pack
+```
```

Comment:
You don't need to tun `npm pack` the build script already do that, there is a `postbuild` hook indeed.

Reply:
Thanks. Did not pay attention to it